modules/lang-mustache/src/main/java/org/opensearch/script/mustache/SearchTemplateResponse.java

```diff
@@ -120,7 +120,10 @@ public static SearchTemplateResponse fromXContent(XContentParser parser) throws
     @Override
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
         if (hasResponse()) {
-            response.toXContent(builder, params);
+            builder.startObject();
```

Comment:
:+1:

Reply:
Hm ... Looking closely, we do actually change schema here. The `SearchResponse` was serialized to / from XContent as-is, but now we add this additional property `status` that we serialize along with it. I think this is fine but needs documentation / spec update :100%